query/templates/search/v710/distinctItemCountQuery.tmpl

```diff
@@ -0,0 +1,19 @@
+{ {{/* count query */}}
+{{if .CountEnable}}
+"query" : {
+    "bool" : {
+      "must" : [{
+                     {{- template "coreQuery.tmpl" .}}
```

Comment:
Can we be consistent with indentation to demonstrate nested objects - this will make it easier to read

Reply:
Fixed in update 1.